### PR TITLE
Motr terminology upload

### PR DESCRIPTION
**Pull request by swatid-seagate**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered Motr-Terminology-Guide.md](https://github.com/swatid-seagate/cortx-motr/blob/Motr-Terminology-upload/Motr-Terminology-Guide.md)

### CODE REVIEW

File: Motr-Terminology-Guide.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+  </tr>
+
+
+
+  <tr>
+      <td>Segment</td>
+      <td>A segment is a stob mapped to an extent in process address space. Each address in the extent uniquely corresponds to the offset in the stob and vice versa. Stob is divided into blocks of fixed size.
+      </td>
+  </tr>
+
```
Comment (mehjoshi) [inline Motr-Terminology-Guide.md:356]: Remove extra blank line.

File: Motr-Terminology-Guide.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+  <tr>
+      <td>Unit Test (UT)</td>
+      <td>UT is a way of testing a smallest piece of code that can be logically isolated in a system.
+      </td>
+  </tr>
+
+
```
Comment (mehjoshi) [inline Motr-Terminology-Guide.md:388]: Remove extra blank line.

File: Motr-Terminology-Guide.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+      </td>
+  </tr>
+
+  <tr>
+      <td>SNS</td>
+      <td>Server Network Striping
+      </td>
+  </tr>
+
+
```
Comment (mehjoshi) [inline Motr-Terminology-Guide.md:349]: Remove extra blank lines.

File: Motr-Terminology-Guide.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+      </td>
+  </tr>
+
+  <tr>
+      <td>Key order</td>
+      <td>Total order, defined on keys within a given container. Iterating through the container, returns keys in this order. The order is defined as lexicographical order of keys, interpreted as bit-strings.
+      </td>
+  </tr>
+
+
```
Comment (mehjoshi) [inline Motr-Terminology-Guide.md:275]: Remove extra blank lines.

File: Motr-Terminology-Guide.md [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+  </tr>
+
+  <tr>
+      <td>Global object layout</td>
+      <td>A global object layout specifies a location of its data or redundancy information as a pair (component-id, component-offset). The component-id is the FID of a component stored in a certain container. (On the components FIDs, the layouts are introduced for the provided global object. But these are not important for the present specification.)
+      </td>
+  </tr>
+
+
+
```
Comment (mehjoshi) [inline Motr-Terminology-Guide.md:237]: Remove extra blank lines.

File: Motr-Terminology-Guide.md [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+  <tr>
+      <td>HA</td>
+      <td>CORTX-HA or High-Availability always ensures the availability of CORTX and prevents hardware component or software service failures. If any of your hardware components or software services are affected, CORTX-HA takes over the failover or failback control flow and stabilizes them across the CORTX cluster.
+      </td>
+  </tr>
+
+  <tr>
+      <td>HA interface</td>
+      <td>HA interface API that allows Halon to control Motr and allows Motr to receive cluster state information from Hare.
```
[the remark this answers is not in the document]
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

File: Motr-Terminology-Guide.md (hunk 1 above)
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md:290]: Done

File: Motr-Terminology-Guide.md (hunk 2 above)
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

File: Motr-Terminology-Guide.md (hunk 3 above)
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

File: Motr-Terminology-Guide.md (hunk 4 above)
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

File: Motr-Terminology-Guide.md (hunk 5 above)
Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

**Comment by mehjoshi** [comment]
@swatid-seagate, Something is not correct with the syntax. I am not able to render the file correctly in a .md file viewer. Please check.

File: Motr-Terminology-Guide.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+    <tr>
+    <th width=220px align="left">Terminology</th>
+    <th width=500px align="left">Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+  <tr>
+    <td>ADDB</td>
+    <td>Analysis and Diaganostics DataBase contains information describing ongoing activity of Motr system</td>
```
Comment (nkommuri) [inline Motr-Terminology-Guide.md:14]: Please correct Diaganostics spelling

Comment (swatid-seagate) [reply Motr-Terminology-Guide.md]: Done

**Review by mehjoshi — APPROVED** [review]

**Review by nkommuri — APPROVED** [review]

**Pull request merged**